Octokit/Models/Response/UserType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Octokit
+{
+    public enum UserType
```

Comment:
I'm not so sold on this name. `AccountType` is used for a certain request, but `Org` doesn't map to `Organization`.